ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml

```diff
@@ -70,6 +70,33 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-preserve-bootstrap-ign
+- as: aws-ipi-shared-phz-f14
+  cron: 14 0 5,19 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - ref: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz
+- as: aws-ipi-shared-phz-sts-f14
+  cron: 27 12 2,18 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - ref: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-sts
+- as: aws-usgov-ipi-private-fips-ep-f14
```

Comment:
Maybe we can move this job to regular openshift ci yaml file?

Reply:
Updated